### DISCUSSION TITLE
AI方向论文笔记模板

### DISCUSSION THREAD

**Discussion by YTZ01** [windingwind/zotero-better-notes#715] [Note Templates]
![d31ca7024769f0f81c7b99575227c4d](https://github.com/windingwind/zotero-better-notes/assets/111220703/80990a2d-510b-49ad-bc33-4c1058f371a9)
```
<span style="color:#5686bf"><h1><font size="6">论文笔记✍</font></h1></span>
<span style="color:#2F2F4F">📅<strong>发表时间：<br/>
<span style="color:#2F2F4F"><strong>🔢期刊会议：<br/>
</span >🎯方向分类</strong>：
</a><br/>
</span></h1>\n <h2 style="color:#ba55dc; background-color:  #fff8dc;">📜 研究内容</h2> <hr/>
<p></p>
<h3>  <span style="color: #e0ffff; background-color:    #66cdaa;">⚙️ 做了什么：</span></h3>
<p></p>
<h3>  <span style="color: #004d99; background-color:    #87cefa;">🔨 解决了什么问题：</span></h3>
<p></p>
</span></h1>\n <h2 style="color:#808080; background-color:    #dcdcdc;">🔬 现状不足</pre></h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#ffd700; background-color:    #fffdd0;">✨ 创新点</pre></h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#fffaf0; background-color:    #ccccff;">🔁 步骤简述</pre></h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#20b2aa; background-color:    #afeeee;">👩🏻‍💻 模型架构</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">🙋‍♀️ 个人问题</h2> <hr/>
<p></p>
<h3>  <span style="color: #de3163; background-color:    #ffc0cb;">🤔 生词扫盲：</span></h3>
<p></p>
<h3>  <span style="color: #da70d6; background-color:    #EED2EE;">📌 待解决点：</span></h3>
<p></p>
</span></h1>\n <h2 style="color:#FF8C00; background-color:   #FFE7BA;">💭 思考启发</h2> <hr/>
```
以个人审美在[windingwind](https://github.com/windingwind/zotero-better-notes/discussions/599)基础上修改的莫兰迪配色AI方向论文阅读笔记模板，小伙伴们可在拙作基础上进行调整~
调整emoji表情：https://www.emojiall.com/zh-hans
调整配色：https://encycolorpedia.cn/named

**Comment by GritYolo** [comment]
your design is so good! Could I get your code?

**Reply by YTZ01** [reply]
Hi there! I've updated the code, thanks for ur appreciation :)

**Comment by yvonmoon** [comment]
感谢！！这是我唯一一个直接成功的模板！

**Reply by Maxine-1520** [reply]
你好，请问是怎么创建的呢？我复制了上述内容到新建模板里，显示invalid

**Reply by wang-zhuoran** [reply]
一样哈哈哈，这也是我唯一一个不用修bug直接成功导入的模板

**Comment by wbd2** [comment]
十分感谢！！！

**Comment by shaoxh** [comment]
我通过“从剪切板导入模板”，它提示 template is invalid。请问是为什么啊？我直接复制的上面的。

**Reply by windingwind** [reply]
本讨论中作者并未使用用于分享的template share code，而是直接粘贴了模板原文。

@tongtong429 建议使用template share code，方便用户导入

<img width="792" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/6d4434a8-2cf9-4bac-ab5e-c95afa7d6f5a">

**Comment by wendribss** [comment]
非常感谢！好用！

**Comment by abandonnnnn** [comment]
太好了！感谢！

**Comment by huqianshan** [comment]
系统自带的模板也会报错，是怎么回事呢

**Comment by ADuStar1y** [comment]
请问为什么我点击从剪切板导入会提示我iinvalid

**Comment by iVincentSan** [comment]
您好，我冒昧使用了您的作品，重新做了一个万能通用的模板，重新写了html。在此向你表达感谢。如有不妥，我会及时删除。
https://github.com/windingwind/zotero-better-notes/discussions/1442

**Comment by jiageling** [comment]
选择新建模板，然后将代码复制到中间的编辑框就可以用了

**Comment by GritYolo** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    谢谢您的回复！
    
    
      
    
        
            
    
        
            
                    
                        
                    
                    
                        ttr326245185
                    
            
                
                    
                            ***@***.***
                    
                
        
    

        
    
 
    

  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2025年11月13日 10:13
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [windingwind/zotero-better-notes] AI方向论文笔记模板 (Discussion #715)
        
      
    
  
  
选择新建模板，然后将代码复制到中间的编辑框就可以用了
